test/Integration/test-java/host.json

```diff
@@ -8,7 +8,7 @@
       }
     },
     "logLevel": {
-      "default": "Information"
+      "default": "Trace"
```

Comment:
There is also the host.json in samples-java which is currently set to "Information".